lib/koans/05_maps.ex

```diff
@@ -14,8 +14,8 @@ defmodule Maps do
   end
 
   koan "A map has keys and values" do
```

Comment:
Seems fine. It's only point is to introduce folks to some of the `Map` interface. Not like we're going to cover it all I suppose.